code/game/objects/structures/stool_bed_chair_nest/alien_nests.dm

```diff
@@ -9,54 +9,47 @@
 	var/image/nest_overlay
 	comfort = 0
 
-/obj/structure/stool/bed/nest/New()
-	nest_overlay = image('icons/mob/alien.dmi', "nestoverlay", layer=MOB_LAYER - 0.2)
-	return ..()
-
-/obj/structure/stool/bed/nest/user_unbuckle_mob(mob/living/user)
-	if(buckled_mob && buckled_mob.buckled == src)
-		var/mob/living/M = buckled_mob
+/obj/structure/stool/bed/nest/user_unbuckle_mob(mob/living/buckled_mob, mob/living/user)
+	if(has_buckled_mobs())
+		var/mob/living/M = buckled_mobs[1]
 
 		if(user.get_int_organ(/obj/item/organ/internal/xenos/plasmavessel))
-			unbuckle_mob()
+			unbuckle_mob(M)
 			add_fingerprint(user)
 			return
 
 		if(M != user)
-			M.visible_message(\
-				"<span class='notice'>[user.name] pulls [M.name] free from the sticky nest!</span>",\
-				"<span class='notice'>[user.name] pulls you free from the gelatinous resin.</span>",\
-				"<span class='notice'>You hear squelching...</span>")
-
+			M.visible_message("[user] pulls [M] free from the sticky nest!",\
+				"<span class='notice'>[user] pulls you free from the gelatinous resin.</span>",\
+				"<span class='italics'>You hear squelching...</span>")
 		else
-			buckled_mob.visible_message(\
-				"<span class='warning'>[buckled_mob.name] struggles to break free of the gelatinous resin...</span>",\
-				"<span class='warning'>You struggle to break free from the gelatinous resin... (This will take around 2 minutes and you need to stay still)</span>",\
-				"<span class='notice'>You hear squelching...</span>")
+			M.visible_message("<span class='warning'>[M] struggles to break free from the gelatinous resin!</span>",\
+				"<span class='notice'>You struggle to break free from the gelatinous resin... (Stay still for two minutes.)</span>",\
+				"<span class='italics'>You hear squelching...</span>")
 			if(!do_after(M, 1200, target = src))
 				if(M && M.buckled)
-					to_chat(M, "<span class='warning'>You fail to escape \the [src]!</span>")
+					to_chat(M, "<span class='warning'>You fail to unbuckle yourself!</span>")
 				return
 			if(!M.buckled)
 				return
-			M.visible_message(\
-			"<span class='warning'>[M.name] breaks free from the gelatinous resin!</span>",\
-			"<span class='notice'>You break free from the gelatinous resin!</span>",\
-			"<span class='italics'>You hear squelching...</span>")
-		unbuckle_mob()
-		add_fingerprint(user)
+			M.visible_message("<span class='warning'>[M] breaks free from the gelatinous resin!</span>",\
+				"<span class='notice'>You break free from the gelatinous resin!</span>",\
+				"<span class='italics'>You hear squelching...</span>")
 
+			unbuckle_mob(M)
+			add_fingerprint(user)

```

Comment:
above two line indented wrong leading to other people being unable to free someone from an alien nest